ProjectManagmentSystem/package.json

```diff
@@ -10,8 +10,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
     "@fontsource/inter": "^5.0.18",
     "@fortawesome/fontawesome-free": "^6.5.2",
+    "@mui/material": "^5.15.17",
```

Comment:
We are not using MUI in this project, we should remove this dependency and any other packages related to it.

Reply:
Removed this was an old pull request so it is still there